docs/src/pages/[platform]/connected-components/authenticator/NextSteps.tsx

```diff
@@ -23,9 +23,9 @@ export function NextSteps() {
       />
       <CardLink
         icon={<MdFlipToFront />}
-        title="Headless usage"
-        href={`/${platform}/connected-components/authenticator/headless`}
-        desc="How to run the Authenticator in a headless mode"
+        title="Advanced usage"
```

Comment:
![image](https://user-images.githubusercontent.com/6165315/229938597-332fa2b4-4162-4957-bbe4-401885d94652.png)

Reply:
Nit: prefer capitalization for the first and principle words of a title, for example:
- "Advanced Usage" 
- "Using the Authenticator For Advanced Use Cases" (note the lowercase "the")

Reply:
fixed